labs/sysdev-tinysystem/sysdev-tinysystem.tex

```diff
@@ -126,6 +126,16 @@ \section{Booting the system}
 error messages in the console. If this doesn't give any clue, you can
 also have a look at the NFS server logs in \code{/var/log/syslog}.
 
+You must allow access to the NFS server through the firewall:
+\begin{verbatim}
+sudo ufw allow nfs
+\end{verbatim}
+
+If this doesn't help, try enabling all traffic with the board:
+\begin{verbatim}
+sudo ufw allow from 192.168.0.100
+\end{verbatim}
```

Comment:
This change seems like a good idea, but I don't like the "If this doesn't help...". Either we provide something that really works, or we fix it to provide something that works.

Also what about TFTP access in the bootloader lab ? Does it need to be enabled as well ?

Reply:
Agreed.